src/hotspot/share/prims/jvmtiUtil.hpp

```diff
@@ -57,7 +57,7 @@ class JvmtiUtil : AllStatic {
       return _event_threaded[num];
     }
     if (num >= EXT_MIN_EVENT_TYPE_VAL && num <= EXT_MAX_EVENT_TYPE_VAL) {
-      return (num != EXT_EVENT_CLASS_UNLOAD);
+      return (num != EXT_EVENT_CLASS_UNLOAD && num != EXT_EVENT_CRAC_BEFORE_CHECKPOINT && num != EXT_EVENT_CRAC_AFTER_RESTORE);
```

Comment:
Any idea why `EXT_EVENT_VIRTUAL_THREAD_MOUNT`/`UNMOUNT` are not listed here?

Reply:
This function checks if an event is related to a particular thread. This is `true` only for `VIRTUAL_THREAD_MOUNT/UNMOUNT`. For both `EXT_EVENT_CLASS_UNLOAD` and for CRaC events it doesn't make sense - so `false` is returned.